performance_test/src/experiment_configuration/experiment_configuration.cpp

```diff
@@ -325,6 +361,34 @@ bool ExperimentConfiguration::is_with_security() const
   return m_with_security;
 }
 
+ExperimentConfiguration::RoundTripMode ExperimentConfiguration::roundtrip_mode() const {
+  check_setup();
+  return m_roundtrip_mode;
+}
+
+std::string ExperimentConfiguration::pub_topic_postfix() const {
+  check_setup();
+  std::string fix;
+  if(m_roundtrip_mode == ExperimentConfiguration::RoundTripMode::MAIN) {
+    fix = "main";
+  }
+  else if(m_roundtrip_mode == ExperimentConfiguration::RoundTripMode::RELAY) {
+    fix = "relay";
+  }
+  return fix;
+}
+
+std::string ExperimentConfiguration::sub_topic_postfix() const {
+  check_setup();
+  std::string fix;
+  if(m_roundtrip_mode == ExperimentConfiguration::RoundTripMode::MAIN) {
+    fix = "relay";
+  }
+  else if(m_roundtrip_mode == ExperimentConfiguration::RoundTripMode::RELAY) {
+    fix = "main";
+  }
+  return fix;
```

Comment:
If roundtrip mode is `NONE`, returns an empty string, so we don't append the topic names correct?

Reply:
Exactly, the behavior does not change if round trip mode is disabled.